.github/workflows/auto-update.yml

```diff
@@ -97,6 +99,43 @@ jobs:
         git commit -m "$MSG"
         git remote set-url origin https://x-access-token:${{ secrets.BOT_TOKEN }}@github.com/$ORG/$BTP_MANAGER_REPO.git
         git push --set-upstream origin $BRANCH_NAME -f
-        gh pr create -B main --title "$MSG" --body ""
+        pr_link=$(gh pr create -B main --title "$MSG" --body "" | tail -n 1)
```

Comment:
Could we add a comment here, that this whole section is about creating notes on Gophers Board?